postprocess.h

```diff
@@ -203,6 +207,116 @@ class PostprocessImpl<ReLU, CPUType::AVX512BW> {
   }
 };
 
+#endif
+
+/*
+ * ReLU_int8
+ */
+class ReLU_int8 {};
+
+template <>
+class PostprocessImpl<ReLU_int8, CPUType::SSE2> {
+public:
+  using InputRegister = RegisterPair128i;
+  using OutputRegister = RegisterPair128i;
+
+  PostprocessImpl(const ReLU_int8& config) {}
+
+  INTGEMM_SSE2 inline OutputRegister run(InputRegister input, Index offset) {
+    static const auto const_zero = setzero_si<__m128i>();
+    return {
+      _mm_and_si128(_mm_cmplt_epi8(const_zero, input.pack0123), input.pack0123),
```

Comment:
Given that we're going to be repacking 32 bits to 8 bits, we may as well have this run on one register rather than two.

Reply:
I changed it.